### PR TITLE
Langchain handler modifications

### PR DESCRIPTION
**Pull request by abhayasr**
## Description

-Stores the input and output of the llm in langchain_handler.py
-Use sqlite to be able to store both the input and the output of each query
-Added logs for when self improvement functionality works

Fixes #issue_number

## Type of change

(Please delete options that are not relevant)

- [ ] ⚡ New feature (non-breaking change which adds functionality)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by abhayasr** [comment]
@paxcema @TylerSandman

File: mindsdb/integrations/handlers/langchain_handler/langchain_handler.py [hunk 1]
```diff
@@ -431,6 +438,8 @@ def _invoke_agent_executor_with_prompt(agent_executor, prompt):
             return answer['output']
 
         completions = []
+        #save the inputs too
+        inputs = []
```
[the remark this answers is not in the document]
Comment (abhayasr) [reply mindsdb/integrations/handlers/langchain_handler/langchain_handler.py:442]: Yes you are right, I don't know why it was down there, I was playing around with some changes seems like I kept it in the wrong place

File: config.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
[the remark this answers is not in the document]
Comment (abhayasr) [reply config.json:1]: I don't know why this file is there, I don't think I added it, I think it was so I was able to launch the web version, next time I will git add specific files I did git add .

File: mindsdb/integrations/handlers/langchain_handler/langchain_handler.py [hunk 3]
```diff
@@ -6,6 +6,9 @@
 import numpy as np
 import pandas as pd
 
+#how to save the output of the llm
+import sqlite3
```
[the remark this answers is not in the document]
Comment (Dav1dF) [reply mindsdb/integrations/handlers/langchain_handler/langchain_handler.py:10]: Yeah, lets have a chat about this whenever you're free

File: config.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (paxcema) [reply config.json:1]: Ah interesting. Well, you can remove specific files like this one with `git rm /path/to/file`

**Comment by abhayasr** [comment]
This is not relavent anymore, the work was already merged on a different branch

**Pull request closed without merging**